dev/src/reference.ts

```diff
@@ -1846,29 +1862,34 @@ export class Query<T = firestore.DocumentData> implements firestore.Query<T> {
     }
```

Comment:
`createImplicitOrderBy` function accepts `DocumentReference` as value for cursor as well, but the server returned result for it is not correct. The cursor parameter type is documented here, it might need further digging.  https://docs.google.com/document/d/1VOJfT4QBOmS8AW8XeWJcpaUdh0dW_RCgs-n9HYTNxTw/edit?resourcekey=0-l27mCN4xSkZLIZx9olzbfw#heading=h.wiam1kg6dphx